### PR TITLE
Sprint7 Lobby Fixes

### PR DESCRIPTION
**Pull request by mchris25**
This is the pull request for Sprint 7 Lobby merging into Sprint 7 Development. It aims to fix the resubmit issue, however the issue which is encountered when matchmaking 2+ teams is still broken due to testing limitations and backend constraints

### CODE REVIEW

**Review by dnguyen3031 — APPROVED** [review]
Fine to merge as is, make sure the leftover bugs are fixed before end of the project

**Review by xhwallace — APPROVED** [review]
Looks good to me

File: end_match.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def assign_elos(lobby):
         compare_other_teams(lobby["teams"][i], lobby["team_info"][i], lobby["team_info"], game["game_name"])
 
 def free_discord(room_name):
-    dis = Discord().find_by_name(room_name)[0]
```
Comment (prcox) [inline end_match.py:49]: change to 0

**Review by jlu51 — CHANGES_REQUESTED** [review]
line 49 of end_match.py should be index 0.

**Review by prcox — APPROVED** [review]
looks good other than that index

**Pull request merged**